keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -280,7 +281,11 @@ So, E2E tests doesn't add extra value to integration tests.
 
 #### Beta (v1.25):
 
-- [ ] This feature will be enabled by default as a Beta feature in v1.25.
+- [x] This feature will be enabled by default as a Beta feature in v1.25.
+
+#### GA (v1.30):
+
+- [x] No particular issue is reported to this feature for a certain length of time.
```

Comment:
Can't comment on unchanged lines, so commenting here:

1. "Are there any tests for feature enablement/disablement?"

Please fill in. In particular, this comment from the KEP template is relevant:
```
Additionally, for features that are introducing a new API field, unit tests that
are exercising the `switch` of feature gate itself (what happens if I disable a
feature gate after having objects written with the new field) are also critical.
You can take a look at one potential example of such test in:
https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
```

Reply:
Actually, we missed such test and we only did the manual testing for switching the feature gate.
I created the PR to add the integration test in https://github.com/kubernetes/kubernetes/pull/123115. However if we will remove the feature gate when reaching GA, the PR is no longer needed.

Reply:
Please update with the text like this then [agree it's too late now...]

Reply:
@wojtek-t Fixed 🙏